docs/source/WritingTests.rst

```diff
@@ -1190,29 +1190,31 @@ tests can do and also they can modify the test parameters.
-+-------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
-| *                       | All variables from --mux-yaml         | TIMEOUT=60; IO_WORKERS=10; VM_BYTES=512M; ...                                                       |
-+-------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| Environemnt Variable        | Meaning                               | Example                                                                                             |
++=============================+=======================================+=====================================================================================================+
+| AVOCADO_VERSION             | Version of Avocado test runner        | 0.12.0                                                                                              |
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| AVOCADO_TEST_BASEDIR        | Base directory of Avocado tests       | $HOME/Downloads/avocado-source/avocado                                                              |
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| AVOCADO_TEST_DATADIR        | Data directory for the test           | $AVOCADO_TEST_BASEDIR/my_test.sh.data                                                               |
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| AVOCADO_TEST_WORKDIR        | Work directory for the test           | /var/tmp/avocado_Bjr_rd/my_test.sh                                                                  |
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| AVOCADO_TEST_SRCDIR         | Source directory for the test         | /var/tmp/avocado_Bjr_rd/my-test.sh/src                                                              |
++-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+| AVOCADO_TESTS_COMMON_TMPDIR | Tests shared temporary directory      | /var/tmp/avocado_XhEdo/                                                                             |
```

Comment:
Nice update, would you please extend the description to emphasize it's persistent directory throughout all tests execution and that it comes from `teststmpdir` plugin? Or do you think it would be too much information?